File: spark-rest-api/src/main/java/com/ssen/GreetingResource.java
package com.ssen;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("/")
public class GreetingResource {

    @GET
    @Path("hello")
    @Produces(MediaType.APPLICATION_JSON)
    public String hello() {
        return "Hello Youtube";
    }
}

File: spark-rest-api/src/main/scala/org/ssen/GreetingResource.scala
package org.ssen

import javax.ws.rs.{GET, Path, Produces}
import javax.ws.rs.core.MediaType

@Path("/hello")
class GreetingResource {

    @GET
    @Produces(Array[String](MediaType.APPLICATION_JSON))
    def hello() = "Hello RESTEasy"
}